crates/cli/src/util.rs

```diff
@@ -264,8 +272,7 @@ Please log back in with `spacetime logout` and then `spacetime login`."
     })
 }
 
-pub fn decode_identity(config: &Config) -> anyhow::Result<String> {
-    let token = config.spacetimedb_token_or_error()?;
+pub fn decode_identity(token: &String) -> anyhow::Result<String> {
```

Comment:
this function now takes a `token` directly, because otherwise it was going to need a bunch of extra params to pass to `get_login_token_or_log_in` and it made a bunch of stuff more messy.